vendor/engines/secure_rooms/spec/features/scans_controller_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "rails_helper"
+
+RSpec.describe "Scanning in and out", type: :controller do
```

Comment:
I feel like if it's acting as a feature spec, it should be in `spec/features`, even if it's using controller tooling rather than capybara. Functionally, this is a feature spec.

This seems like an important spec to have, but if I saw this is `spec/controllers` I'd be pretty confused, because this doesn't follow a controller spec structure.

What if you just added a comment in this file explaining that this is intended as a feature spec, but you are using controller tooling because it's hard to test APIs with capybara?